src/Auth0Client.ts

```diff
@@ -719,6 +747,7 @@ export default class Auth0Client {
 
     const tokenResult = await oauthToken(
       {
+        ...this.customOptions,
         ...customOptions,
```

Comment:
I wonder if it's worth renaming the local `customOptions` here, alongside `this.customOptions`, it could be confusing as to why they're both there. Or do you feel the reference to `this` is enough to differentiate them?